src/oneid/oneid-ecs-core/src/main/resources/application.properties

```diff
@@ -1,6 +1,9 @@
-%dev.quarkus.dynamodb.devservices.enabled=false
-%dev.quarkus.dynamodb.endpoint-override=http://localhost:8000
+quarkus.dynamodb.devservices.enabled=false
+quarkus.dynamodb.aws.region=local
+quarkus.dynamodb.aws.credentials.type=static
+#Dev
 %dev.quarkus.dynamodb.aws.region=local
```

Comment:
maybe lines `5` and `6` are redundant with lines `2` and `3`.

Consider removing `5` and `6`

Reply:
Sorry, I understood your point, you can resolve the conversation!